controllers/replication-controller/persistentvolumeclaim_controller.go

```diff
@@ -144,6 +147,32 @@ func (r *PersistentVolumeClaimReconciler) Reconcile(ctx context.Context, req ctr
 	remoteClaim := &v1.PersistentVolumeClaim{}
 	remotePVCName := ""
 	remotePVCNamespace := ""
+	if remoteClusterID != controller.Self && r.AllowPVCCreationOnTarget {
```

Comment:
In case of single/stretched cluster, what is the plan for target/replicated PV ?

Reply:
Single cluster the replicated pv will be in reserved state with claimref details updated. PVC can be created manually if needed .